src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-23931"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {}
```

Comment:
@anurse what would help, this is not tied to this PR *per se* but overall, is a matrix of what is gotten from each TFM. Something like this:

| Package               	| netcoreapp1.0    	| netstandard1.5 	|
|-----------------------	|------------------	|----------------	|
| Microsoft.NETCore.App 	| Core console app 	| ???            	|
| Netstandard.Library   	| Self-contained   	| PCL            	|